pkg/client/v1/model/initiator_group.go

```diff
@@ -1,35 +1,70 @@
-/**
- * Copyright 2017 Hewlett Packard Enterprise Development LP
- */
+// Copyright 2020 Hewlett Packard Enterprise Development LP
 
 package model
-//package nimblestorage/v1/InitiatorGroup
 
 
-// InitiatorGroup :
+// InitiatorGroup - Manage initiator groups for initiator authentication. An initiator group is a set of initiators that can be configured as part of your ACL to access a specific volume through group membership.
+// Export InitiatorGroupFields for advance operations like search filter etc.
+var InitiatorGroupFields *InitiatorGroup
+
+func init(){
+	IDfield:= "id"
+	Namefield:= "name"
+	FullNamefield:= "full_name"
+	SearchNamefield:= "search_name"
+	Descriptionfield:= "description"
+	HostTypefield:= "host_type"
+	AppUuidfield:= "app_uuid"
+		
+	InitiatorGroupFields= &InitiatorGroup{
+	ID: &IDfield,
+	Name: &Namefield,
+	FullName: &FullNamefield,
+	SearchName: &SearchNamefield,
+	Description: &Descriptionfield,
+	HostType: &HostTypefield,
+	AppUuid: &AppUuidfield,
```

Comment:
indent

Reply:
I tried to indent but freemarker doesnt have rich syntaxes to do that.

Reply:
Done.